core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -287,9 +287,6 @@ object AppType {
   def stringToObject: Map[String, AppType] = values.map(v => v.toString -> v).toMap
 }
 
-final case class ReleaseName(value: String) extends AnyVal
-final case class ChartName(value: String) extends AnyVal
-
```

Comment:
couldn't see why we'd want to duplicate `Chart{Name,Version}` and `Release` which exist in `helm-scala-sdk`